drivers/hmis/spec/requests/hmis/client_enrollments_performance_spec.rb

```diff
@@ -105,9 +92,10 @@
 
     it 'minimizes n+1 queries' do
       expect do
-        _, result = post_graphql(**variables) { query }
+        response, result = post_graphql(**variables) { query }
+        expect(response.status).to eq(200), result.inspect
         expect(result.dig('data', 'client', 'enrollments', 'nodes').size).to eq(enrollments.size)
-      end.to make_database_queries(count: 10..30)
+      end.to make_database_queries(count: 10..35)
```

Comment:
This increased due to https://github.com/greenriver/hmis-warehouse/pull/3682/files#r1378003488 , I think. Would definitely like to fidn a way to bring it down

Reply:
It's just a few extra queries per client right? That doesn't seem so bad considering the granular permissions we are introducing.